src/main/java/org/glassfish/grizzly/thrift/client/pool/BaseObjectPool.java

```diff
@@ -718,14 +728,8 @@ public Builder(PoolableObjectFactory<K, V> factory) {
 
     @Override
     public String toString() {
-        return "BaseObjectPool{" +
```

Comment:
Previous better

Reply:
It's a simple change, so I think it'll be fine.